pkg/util/kubernetes/autoscalers/datadogexternal.go

```diff
@@ -141,6 +172,34 @@ func (p *Processor) queryDatadogExternal(metricNames []string) (map[string]Point
 	return processedMetrics, nil
 }
 
+func (p *Processor) updateRateLimiting() error {
+	updateMap := p.datadogClient.GetRateLimitStats()
+	queryLimits := updateMap[queryEndpoint]
+
+	var errors []error
+
+	lim, errLim := strconv.Atoi(queryLimits.Limit)
```

Comment:
Might be good to use a helper here - repeated conversion code for all four below.